wire-ios-sync-engine/Tests/Source/Calling/WireCallCenterV3Tests.swift

```diff
@@ -68,14 +68,16 @@ class WireCallCenterV3Tests: MessagingTest {
     override func setUp() {
         super.setUp()
 
+        BackendInfo.domain = "wire.com"
```

Comment:
we should avoid changing this directly/alone. change the storage before so it does not affect other tests
```
BackendInfo.storage = UserDefaults(suiteName: UUID().uuidString)!
```

Reply:
fixed